src/v/cluster/rm_partition_frontend.cc

```diff
@@ -673,4 +673,67 @@ ss::future<abort_tx_reply> rm_partition_frontend::do_abort_tx(
       });
 }
 
+ss::future<tx_errc> rm_partition_frontend::force_abort_tx(
```

Comment:
`rm_partition_frontend` describes data partition's internal interface which is used by txn coordinator. an abort invoked by user doesn't fit that scenario so no need to modify rm_partition_frontend & tx_gateway_frontend.

use an approach similar to src/v/kafka/server/handlers/produce.cc and src/v/redpanda/admin_server.cc::mark_transaction_expired_handler: query partition_manager for a partition, get rm_stm and invoke mark_expired